### ISSUE TITLE
Notify when user loads incompatible trace file

### ISSUE THREAD

**Issue by ebugden** [theia-ide/theia-trace-extension#328]
Currently there's no feedback to the user if they load a file that isn't a trace or an incompatible trace file. This confused user test participants. There needs to be an error message and the files should not be added to the list of traces.

**Possible solution (popping a notification):**
![image](https://user-images.githubusercontent.com/28311615/118338028-236f3a80-b4e3-11eb-8822-2ec52cc630e7.png)

Message needs to be nuanced: Consider that the file could be a valid trace file, just a format not yet supported. (ex. Unable to open (Incompatible or invalid trace))

----
Issue source: @ssmagula's user tests + brainstorming with TC and tool team members

**Comment by IbrahimFradj** [comment]
I just tried to open a file (not a trace) with trace viewer and it notifies me that it is not a valid trace. 

**Here is the image of the notification.**

![image](https://user-images.githubusercontent.com/83769103/118701143-641bcc00-b7e1-11eb-8268-4e01abae94e8.png)

**Comment by ebugden** [comment]
Ok thanks to git blame I discovered this was fixed by #231 (two months before this issue was even created...). According to a PR [comment](https://github.com/theia-ide/theia-trace-extension/pull/231#issuecomment-764884414) it adds:
1. Show an error message in case all the imported traces are invalid.
2. Show a warning message instead if 'any but not all' the traces are invalid and display the list of invalid traces in the toast message.

----

I'm curious to know how this issue came up during @ssmagula's user tests (which were in Feb?), maybe an older version of the tool was used? Was the error not obvious enough? Did the user test catch a corner case and no message showed?
